### PR TITLE
Update install-minikube.md

### PR DESCRIPTION
**Pull request by ryanwyu**
modify the minikube start parameter for driver

### CODE REVIEW

**Comment by RA489** [comment]
@ryanwyu Minikube has changed `--vm-driver` options to `--driver `option.

File: content/en/docs/tasks/tools/install-minikube.md [hunk 1]
```diff
@@ -219,7 +219,7 @@ For setting the `--driver` with `minikube start`, enter the name of the hypervis
 {{< /note >}}
 
 ```shell
-minikube start --driver=<driver_name>
```
Comment (RA489) [inline content/en/docs/tasks/tools/install-minikube.md:222]: This is correct as minikube has deprecated --vm-driver flag.

**Comment by sftim** [comment]
Thanks for the PR @ryanwyu!

Before it can be reviewed, you need to sign the [CLA](https://github.com/kubernetes/website/pull/21104#issuecomment-631991540).

/close

You are welcome to sign the project CLA and then open a new PR based on the same branch.

**Pull request closed without merging**